### ISSUE TITLE
Unable to open Dbeaver after updating to 7.3.2-1

### ISSUE THREAD

**Issue by ricardofpt** [dbeaver/dbeaver#11092]
#### System information: 
- Manjaro Linux
- 7.3.2-1
- 1.8.0_275
- Office plugin & Apache POI plugin

#### Connection specification:
- Database name and version - none
- Driver name - none
- Do you use tunnels or proxies (SSH, SOCKS, etc)? - no

#### Describe the problem you're observing:
After updating dbeaver to the version in the title (I used the system's package manager as always), every time I try to open dbeaver I get a small error window: 
![Screenshot at 2021-01-22 11-57-54](https://user-images.githubusercontent.com/18679158/105488259-40b89880-5ca9-11eb-888b-8c9746239cbc.png)

From the log it seems that some eclipse modules are gone...has this happened to anyone?

#### Include any warning/errors/backtraces from the logs
[1611316665339.log](https://github.com/dbeaver/dbeaver/files/5855574/1611316665339.log)

**Comment by conao3** [comment]
Same here, I downgrade dbeaver to `7.3.0-1`, works fine.

**Comment by ricardofpt** [comment]
Thank you for your comment, I guess I'll do that for now.

Hope that this gets fixed meanwhile :)

**Comment by conao3** [comment]
I hope so too. But this project is too big so I can only report this issue...
This is what I see and its log.
## SS
![Screenshot_2021-01-23_02-49-20](https://user-images.githubusercontent.com/4703128/105526474-d1cd5500-5d25-11eb-897a-11c995f7c937.png)

## Log
https://gist.github.com/conao3/9390443dba855901538c4747ee8829b8

**Comment by ricardofpt** [comment]
You have exactly the same log output as me.

I followed your advice and downgraded, and everything is working now :)

**Comment by p91paul** [comment]
did you report downstream to manjaro as well? the bug might also be in packaging

**Comment by conao3** [comment]
Thanks, I opened report at archlinux forum.
https://bugs.archlinux.org/task/69430

**Comment by conao3** [comment]
I confirmed this issue completely packaging issue.
```
$ curl -LO https://dbeaver.io/files/dbeaver-ce-latest-linux.gtk.x86_64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   154  100   154    0     0    221      0 --:--:-- --:--:-- --:--:--   221
100   154  100   154    0     0    178      0 --:--:-- --:--:-- --:--:--   178
100 93.8M  100 93.8M    0     0  7585k      0  0:00:12  0:00:12 --:--:-- 7331k

$ tar xf dbeaver-ce-latest-linux.gtk.x86_64.tar.gz 

$ cd dbeaver/
.   configuration  dbeaver.desktop  dbeaver.png      features  jre       p2       readme.txt
..  dbeaver        dbeaver.ini      .eclipseproduct  icon.xpm  licenses  plugins

$ ./dbeaver
<works fine!>
<'About' says its version is Version 7.3.3.202101161839>
```

**Comment by ricardofpt** [comment]
Ok, that's it then!
Thank you for looking into it @conao3.

Closing the issue.

**Comment by patrickathompson** [comment]
I am running Archlinux and was also seeing this error after upgrade. The issue for me was I was using jdk8 as the default java (for development purposes). jdk11 was already installed and once I used archlinux-java to change the default java version, everything was working fine after that. What gave me a clue to try this was that I noticed a change around the time the issue started happening for me:

https://github.com/archlinux/svntogit-community/commit/8c91aa2e07a2571dcee16515a97af05ab3c2e71d#diff-37538beb61ff63edebbf735dfcf39e5d732f49183d6beb097169d971875ca422R11
